spec/jest_spec.vim

```diff
@@ -16,68 +16,68 @@ describe "Jest"
       view +1 __tests__/normal-test.js
       TestNearest
 
-      Expect g:test#last_command == 'jest __tests__/normal-test.js -t ''^Math'''
+      Expect g:test#last_command == 'jest -t ''^Math'' -- __tests__/normal-test.js'
```

Comment:
Could you please elaborate on the `--` before the test name. I had a look at [the documentation](https://facebook.github.io/jest/docs/en/cli.html) and I can't find an example of this